stdlib/public/SDK/AppKit/AppKit.swift

```diff
@@ -14,6 +14,7 @@ import Foundation
 @_exported import AppKit
 
 extension NSCursor : _DefaultCustomPlaygroundQuickLookable {
+  @available(*, deprecated, message: "NSCursor._defaultCustomPlaygroundQuickLook will be removed in a future Swift version")
```

Comment:
I think we should still allow using it in Swift 3 and Swift 4.1 without warning. Therefore the attribute should look like:
`@available(swift, deprecated: 4.1, message: "...")`.